roles/ealapps/tasks/main.yml

```diff
@@ -25,10 +10,26 @@
     mode: 0775
   changed_when: false
 
+- name: ealapps | create directories for shared files
+  ansible.builtin.file:
+    path: '{{ capistrano_base_dir }}/{{ item }}'
+    state: 'directory'
+    owner: '{{ deploy_user }}'
+    group: '{{ deploy_user }}'
+    mode: 0755
+  changed_when: false
+  loop:
+    - '{{ capistrano_directory }}/shared/EALJ'
+    - '{{ capistrano_directory }}/shared/EALJ/pdfs'
```

Comment:
This directory, or a sub-directory called /shared/EALJ/pdfs/tmp/ needs to have 0666 perms so individual articles can be served up to the user. Add a task for this.